tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2588,3 +2588,18 @@ def test_quantize_dates(self, mock_quantize):
             )
 
             assert len(mock_quantize.mock_calls) == 2
+
+    def test_limit_number_of_fields(self):
+        self.login_as(user=self.user)
+        self.create_project()
+        with self.feature("organizations:discover-basic"):
+            for i in range(1, 40):
```

Comment:
Do you need to go to 40 if the limit is at 20? Would stopping at 25 give you the same coverage that 40 does but spend less run time?